README.md

```diff
@@ -19,3 +19,127 @@ bayernluefter:
 ## Installation
 
 Copy into your custom_components directory or install via [HACS](https://hacs.xyz/)
+
+## The result:
+
+![Configured groups containing Bayernluft informations](bayernluftresult.png)
+![Custombayernluft card](bayernluftcustomcard.png)
+
+The second card which is taken from: https://github.com/arjenvrh/audi_connect_ha requires the following mods: 
+
+https://github.com/thomasloven/lovelace-card-mod
+
+https://github.com/custom-cards/circle-sensor-card
+
+and the following configuration:
+
+```yaml
+type: picture-elements
+image: /local/img/bayernluefter.png
+style: |
+  ha-card {
+    border-radius: 10px;
+    border: solid 1px rgba(100,100,100,0.3);
+    box-shadow: 3px 3px rgba(0,0,0,0.4);
+    background-color: white;
+    overflow: hidden;
+  } 
+elements:
+  - type: image
+    image: /local/img/bayernluefter.png
```

Comment:
Does the user need to copy some file somewhere or is the image included with the custom component?

Reply:
We could include it when we change the link to GitHub and upload the picture. 
I thing it would be better for the user to download the picture and put it in www/img/ folder

Reply:
Okay I agree :)